src/retry.ts

```diff
@@ -103,9 +103,8 @@ function shouldRetryRequest(err: GaxiosError) {
   const config = getConfig(err);
 
   if (
-    (err.config.signal?.aborted && err.error?.name !== 'TimeoutError') ||
-    err.name === 'AbortError' ||
-    err.error?.name === 'AbortError'
+    (err.config.signal?.aborted && err.code !== 'TimeoutError') ||
+    err.code === 'AbortError'
```

Comment:
why are we getting rid of `err.name`?

Reply:
That line could never be true as `err.name` is always `GaxiosError`

Reply:
It’s not relevant, but is a one-line change. Not at all a big deal. Not worthy of a separate PR.

Reply:
Hm, those are often the most tricky! We've had this situation occur before. Until we have very robust integration testing, I think it would be wise to make small incremental changes. If we had good integration tests I'd feel better about it!